src/app/flashcard/flashcard-routing.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { CreateFlashcardPageComponent } from './components/pages/create-flashcard-page/create-flashcard-page.component';
+import { FlashcardPageComponent } from './components/pages/flashcard-page/flashcard-page.component';
+
+
+const routes: Routes = [
+  { path: '', component: FlashcardPageComponent },
+  { path: 'create', component: CreateFlashcardPageComponent }
```

Comment:
We would also want a specific path for the flashcard page so that we can redirect to it 

{ path: 'flashcard', component: FlashcardPageComponent }